libs/MeshKernelApi/include/MeshKernelApi/MeshKernel.hpp

```diff
@@ -408,6 +408,16 @@ namespace meshkernelapi
                                                              int startSplineIndex,
                                                              int endSplineIndex);
 
+        /// @brief Snaps the polygon to the land boundary
+        ///
+        /// @param[in] meshKernelId     The id of the mesh state
+        /// @param[in] land             The land boundary
+        /// @param[in] polygon          The polygon values to be snapped
+        /// @returns Error code
+        MKERNEL_API int mkernel_polygon_snap_to_landboundary(int meshKernelId,
```

Comment:
should we also pass the indices of the polygon vertices defining the portion of the polygon that should be snapped?

Reply:
Added start and end index